org.civicrm.bootstrapcivicrm/js/civihr-popup.js

```diff
@@ -0,0 +1,62 @@
+/*
+Prevent Popups to overflow tables.
+*/
+CRM.$(function($) {
+  'use strict';
+
+  (function($) {
+    var MutationObserver = window.MutationObserver || window.WebKitMutationObserver || window.MozMutationObserver;
+
+    $.fn.attrchange = function(callback) {
+      if (MutationObserver) {
+        var options = {
+          subtree: false,
+          attributes: true
+        };
+
+        var observer = new MutationObserver(function(mutations) {
+          mutations.forEach(function(e) {
+            callback.call(e.target, e);
+          });
+        });
+
+        return this.each(function() {
+          observer.observe(this, options);
+        });
+      }
+    }
+  })($);
+
+  // .btn-slide is a continer of popup and a button triggering it
+  $('.btn-slide').attrchange(function(e) {
+    var $button = $(this),
+        $popup,
+        button_offset;
+
+    if($button.hasClass('btn-slide-active')) {
+      // check if popup is open
+      $popup = $button.children('ul.panel');
+
+      $popup.appendTo('body')
+        .addClass('civihr-popup');
```

Comment:
actually I'm afraid it may cause conflicts. There could be many `ul.panel` elements, not necessarily serving as popups, also this name is not descriptive. I think that adding this class may save us from some troubles in the future.